core/server/master/src/main/java/alluxio/master/file/meta/InodeTree.java

```diff
@@ -373,6 +374,41 @@ public boolean inodeIdExists(long id) {
     return mInodeStore.get(id).isPresent();
   }
 
+  /**
+   * Acquires the corresponding inodes along the given path.
+   * @param path the target alluxio path
+   * @return non-empty list if all inodes are successfully loaded
+   * @throws InvalidPathException handle invalid path when getting components
+   */
+  public List<InodeView> getInodesByPath(String path) throws InvalidPathException {
```

Comment:
where is the `InvalidPathException` thrown? Add a comment on that line so people know how non-existing paths are handled.